### PR TITLE
New check - Two Digit Year Cutoff

### PR DESCRIPTION
**Pull request by ClaudioESSilva**
## Please confirm you have 0 failing Pester Tests

[x] There are 0 failing Pester tests

## Changes this PR brings
 - Added New Check for `Two Digit Year Cutoff` configuration
 - Added `policy.twodigityearcutoff` with default 2049 (SQL Server default value)

Based on my blog post [Don’t cutoff yourself with dates in T-SQL – Did you know…](https://claudioessilva.eu/2018/09/04/dont-cutoff-yourself-when-dealing-with-dates-in-t-sql-did-you-know/) and also because @shaneis (thanks man :+ ) remembered me that this was not part of dbachecks 😄

### CODE REVIEW

File: internal/assertions/Instance.Assertions.ps1 [hunk 1]
```diff
@@ -50,4 +50,11 @@ function Assert-InstanceSupportedBuild {
 		$Results.SupportedUntil | Should -BeGreaterThan $Date -Because "this build $($Results.Build) is now unsupported by Microsoft"
         $Results.SupportedUntil | Should -BeGreaterThan $expected -Because "this build $($results.Build) will be unsupported by Microsoft on $SupportedUntil which is less than $BuildWarning months away"
     }
+}
+function Assert-TwoDigitYearCutoff {
+    Param(
+        [string]$Instance,
+        [int]$TwoDigitYearCutoff
+    )
+    (Get-DbaSpConfigure -SqlInstance $Instance -ConfigName 'TwoDigitYearCutoff').ConfiguredValue | Should -Be $defaulTwoDigitYearCutofftbackupcompression -Because 'This is the value that you have chosen for Two Digit Year Cutoff configuration'
```
Comment (shaneis) [inline internal/assertions/Instance.Assertions.ps1]: Think you have a problem here with your variable name...

Unless it is supposed to be " $defaulTwoDigitYearCutofftbackupcompression" 😂

Comment (SQLDBAWithABeard) [reply internal/assertions/Instance.Assertions.ps1]: I concur :-)

Comment (ClaudioESSilva) [reply internal/assertions/Instance.Assertions.ps1]: It's a new property but is still hidden :sob: 
Thanks @shaneis 
Fixed!

**Comment by ClaudioESSilva** [comment]
@SQLDBAWithABeard can you please help on this one? 🤔

**Pull request merged**